src/mvs/workspace.h

```diff
@@ -63,34 +67,78 @@ class Workspace {
 
   Workspace(const Options& options);
 
-  void ClearCache();
+  // Do nothing when we use a cache. Data is loaded as needed.
+  virtual void Load(const std::vector<std::string>& image_names);
+
+  inline const Options& GetOptions() const { return options_; }
 
-  const Options& GetOptions() const;
+  inline const Model& GetModel() const { return model_; }
 
-  const Model& GetModel() const;
-  const Bitmap& GetBitmap(const int image_idx);
-  const DepthMap& GetDepthMap(const int image_idx);
-  const NormalMap& GetNormalMap(const int image_idx);
+  virtual inline const Bitmap& GetBitmap(const int image_idx) {
```

Comment:
I suggest moving these method definitions to the .cc file. Since it's a virtual method, it won't be able to inline these anyway.